subsys/event_manager_proxy/event_manager_proxy.c

```diff
@@ -288,7 +289,11 @@ static int send_event_to_remote(struct emp_ipc_data *ipc, const struct app_event
 	memcpy(buffer, eh, sizeof(buffer));
 	remote_eh->type_id = remote_ev;
 
-	int ret = ipc_service_send(&ipc->ept, buffer, sizeof(buffer));
+	for (size_t cnt = CONFIG_EVENT_MANAGER_PROXY_SEND_RETRIES; cnt > 0; --cnt) {
```

Comment:
number of tries cannot be zero... `cnt = CONFIG_EVENT_MANAGER_PROXY_SEND_RETRIES + 1`?

Reply:
Done